assets/js/modules/search-console/components/module/ModuleOverviewWidget.js

```diff
@@ -0,0 +1,326 @@
+			datapointUnit: '%',
+			change: averageCTRChange,
+			changeDataUnit: '%',
+			context: 'button',
+			selected: selectedStats === 2,
+			handleStatSelection,
+		},
+		{
+			className: 'googlesitekit-data-block--position googlesitekit-data-block--button-4',
+			title: __( 'Average Position', 'google-site-kit' ),
+			datapoint: averagePosition,
+			change: averagePositionChange,
+			changeDataUnit: '%',
+			context: 'button',
+			selected: selectedStats === 3,
+			handleStatSelection,
+		},
+	];
+
+	let title = __( 'Search Traffic Summary', 'google-site-kit' );
+	if ( currentEntityTitle ) {
+		/* translators: %s: page title */
+		title = sprintf( __( 'Search Traffic Summary for %s', 'google-site-kit' ), decodeHTMLEntity( currentEntityTitle ) );
+	}
+
+	const options = {
+		chart: {
+			title,
+		},
+		curveType: 'line',
+		height: 270,
+		width: '100%',
+		chartArea: {
+			height: '77%',
+			left: 60,
+			width: '100%',
+		},
+		legend: {
+			position: 'top',
+			textStyle: {
+				color: '#616161',
+				fontSize: 12,
+			},
+		},
+		hAxis: {
+			format: 'M/d/yy',
+			gridlines: {
+				color: '#fff',
+			},
+			textStyle: {
+				color: '#616161',
+				fontSize: 12,
+			},
+		},
+		vAxis: {
+			direction: selectedStats === 3 ? -1 : 1,
+			gridlines: {
+				color: '#eee',
+			},
+			minorGridlines: {
+				color: '#eee',
+			},
+			textStyle: {
+				color: '#616161',
+				fontSize: 12,
+			},
+			titleTextStyle: {
+				color: '#616161',
+				fontSize: 12,
+				italic: false,
+			},
+		},
+		series: {
+			0: {
+				color: metrics[ selectedStats ].color,
+				targetAxisIndex: 0,
+			},
+			1: {
+				color: metrics[ selectedStats ].color,
+				targetAxisIndex: 0,
+				lineDashStyle: [ 3, 3 ],
+				lineWidth: 1,
+			},
+		},
+		tooltip: {
+			isHtml: true, // eslint-disable-line sitekit/acronym-case
+			trigger: 'both',
+		},
+		focusTarget: 'category',
+	};
+
+	// Split the data in two chunks.
+	const half = Math.floor( data.length / 2 );
+	const latestData = data.slice( half );
+	const olderData = data.slice( 0, half );
+
+	const googleChartData = getSiteStatsDataForGoogleChart(
+		latestData,
+		olderData,
+		metrics[ selectedStats ].label,
+		metrics[ selectedStats ].metric,
+	);
+
+	const currentDayCount = getCurrentDateRangeDayCount( dateRange );
+
+	return (
+		<Widget
+			noPadding
+			Header={ () => (
+				<Fragment>
+					<WidgetHeaderTitle
+						title={ sprintf(
+							/* translators: %s: number of days */
+							_n( 'Overview for the last %s day', 'Overview for the last %s days', currentDayCount, 'google-site-kit', ),
+							currentDayCount,
+						) }
+					/>
+					<WidgetHeaderCTA
+						href={ searchConsoleDeepLink }
+						label={ sprintf(
+							/* translators: %s: module name. */
+							__( 'See full stats in %s', 'google-site-kit' ),
+							_x( 'Search Console', 'Service name', 'google-site-kit' )
+						) }
+					/>
+				</Fragment>
+			) }
+		>
+			<Grid>
+				<Row
+					role="toolbar"
+					aria-label="Line Chart Options"
```

Comment:
Looks like this ARIA label is irrelevant here. Could you please update it to have the relevant label and make it translatable?

Reply:
@eugene-manuilov I don't think we actually need any label here TBH.